File: JavaScript/3HammingDistance.js
//3. - Hamming distance
var str1 = "patete"
var str2 = "patata"

var distance = 0

if (str1.length == str2.length){

    for (var i = 0; i < str1.length; i++){
        if (str1[i] != str2[i]){
            distance++
        }
    }
    console.log(distance)

} else {
    console.log('different lengths')
}

File: Python/3HammingDistance.py
#3. - Hamming distance
str1 = "patete"
str2 = "patata"

distance = 0

if len(str1) == len(str2):
    for i in range(len(str1)):
        if str1[i] != str2[i]:
            distance+=1
    print(distance)
else:
    print('different lengths')